packages/devtools-core/src/App/UserAppContainer/index.tsx

```diff
@@ -49,20 +57,10 @@ const UserAppContainer = (props: Props) => {
         }
       };
     }
-  }, []);
+  });
```

Comment:
Why remove the empty array? Is this effect supposed to be run whenever any prop changes?

Reply:
When it runs for the first time, styled components aren't ready yet, so they don't get copied over. But you're right, the listeners should only run once - will separate the effects.

Reply:
Are the style nodes going to continuously be added every time this component rerenders? Maybe they should be removed in the return callback of use effect?

Reply:
Just noting here for posterity - the styled components style nodes do keep being added. For now this seems to work fine when I edit stylesheets or styled components. If we run into issues with the future, we should revisit.